cassandane/utils/cyrus-perl-paths.pl

```diff
@@ -32,7 +32,27 @@ sub _cyrus_perl_search_path
     return @paths;
 }
 
-my $cassini = Cassandane::Cassini->instance();
+sub usage
+{
+    printf STDERR "Usage: cyrus-perl-path.pl [--config <cassandane.ini>]\n";
+    exit(1);
+}
+
+my %cassini_params;
+
+while (my $a = shift)
+{
+    if ($a eq '--config')
+    {
+        $cassini_params{filename} = shift;
+    }
+    else
+    {
+        usage;
+    }
+}
+
+my $cassini = Cassandane::Cassini->instance(%cassini_params);
```

Comment:
Since this is a single process script which only ever calls methods directly on its own `$cassini` object, it could bypass `instance()` and call `new()` directly with the params it wants, and then we wouldn't need the awkward racey change to `instance()`

But I think the better solution is just making Cassini->new() able to read a cassandane.ini path from the environment as per my other comment, and then nothing else needs to change at all